metadat/developer/Nintendo - Game Boy Advance.dat

```diff
@@ -12187,15 +12187,15 @@ game (
 )
 
 game (
-	name "Pocket Monsters - Fire Red (Japan)"
+	name "Pocket Monsters - FireRed (Japan)"

```

Comment:
We should likely follow whatever No-Intro gives us. If you want to push this change forwards to them, it will make maintenance a lot easier.